pkgs/tools/misc/grub/default.nix

```diff
@@ -42,8 +40,12 @@ let
   canEfi = lib.any (system: stdenv.hostPlatform.system == system) (lib.mapAttrsToList (name: _: name) efiSystemsBuild);
   inPCSystems = lib.any (system: stdenv.hostPlatform.system == system) (lib.mapAttrsToList (name: _: name) pcSystems);
 
-  version = "2.06";
-
+  gnulib = fetchFromSavannah {
+    repo = "gnulib";
+    # NOTE: keep in sync with bootstrap.conf!
```

Comment:
Can we check this with a grep or so?

Reply:
We probably should, yeah. I'll see if there's an easy way to do it.

Reply:
Done.